revolv/static/project.scss

```diff
@@ -265,6 +268,11 @@ body {
             font-size: 1.8rem;
         }
     }
+
+    .columns {
```

Comment:
Can you make this a different class name? I think that in general applying styles to foundation-specific classes is a little bit like using global variables: easier to understand how things work if you use local support instead :)